index.js

```diff
@@ -6,10 +6,18 @@ const http = require('http');
 const request = require('request');
 
 const ACTIONS = {
-	'startEncoder1':			{ label: 'Start Encoder 1',			apiCommand: 'StartEncoder1' },
-	'startEncoder2':			{ label: 'Start Encoder 2',			apiCommand: 'StartEncoder2' },
-	'stopEncoder1':				{ label: 'Stop Encoder 1',			apiCommand: 'StopEncoder1'  },
-	'stopEncoder2':				{ label: 'Stop Encoder 2',			apiCommand: 'StopEncoder2'  }
+	'startEncoder1':			{ label: 'HDX: Start Encoder 1',				apiCommand: 'StartEncoder1' },
+	'startEncoder2':			{ label: 'HDX: Start Encoder 2',				apiCommand: 'StartEncoder2' },
+	'stopEncoder1':				{ label: 'HDX: Stop Encoder 1',					apiCommand: 'StopEncoder1'  },
+	'stopEncoder2':				{ label: 'HDX: Stop Encoder 2',					apiCommand: 'StopEncoder2'  },
+	'startBothEncoders':		{ label: 'HDX: Start Both Encoders',			apiCommand: 'StartBothEncoders'  },
+	'stopBothEncoders':			{ label: 'HDX: Stop Both Encoders',				apiCommand: 'StopBothEncoders'  },
```

Comment:
@krocheck These two lines would be relevant to the LCS as well as the HDX. At this stage the LCS would have no other commands.

Reply:
Copy that

Reply:
@krocheck I know this has sat for some time now. I have had my integration with the Matrox Monarch HD running weekly, (including polling the device for current status) for more almost 18 months. Is it worth me looking into re-integrating it into this module? (I have some more experience with Node/JS/Companion in the time since), or just leave it for now without apparent demand (no issues raised here since 2019).

Reply:
I am looking for this functionality. @bevanjkay, can you outline how to integrate your fixes for the HD into the Monarch connection in Companion? Thanks for any guidance!

Reply:
@mtietje Do you know how to build Companion from source? You can replace the module with my fork and it should work. https://github.com/bevanjkay/companion-module-matrox-monarch-hd

Reply:
I would also love to have this functionality. Have 2 devices I wish to control now several times a week. Unfortunately I just don't quite have the experience to build from source.

Reply:
@j-mardon I'll look to try to get an up to date PR for this ready soon, but might not be for a couple of weeks.
Feel free to send me an email (in my GitHub profile), and I may be able to get something to you earlier.